app.rb

```diff
@@ -75,18 +64,20 @@ def create_person
     create_person_object(select_person)
   end
 
-  def display_person(person)
-    type = '[Student]'
-    type = '[Teacher]' unless person == @student
+  def display_person
+    person = @student.concat(@teacher)
+    return puts "\n>>>> No Person available<<<" if person.empty?
 
-    return if person.empty?
-
-    person.each { |p| puts "#{type} Name: #{p.name}\t\tID: #{p.id}\t\tAge: #{p.age}" }
+    puts
+    person.each do |p|
+      type = '[Student]'
+      type = '[Teacher]' if p.instance_of?(Teacher)
+      puts "#{type} Name: #{p.name}\t\tID: #{p.id}\t\tAge: #{p.age}"
+    end
   end
 
   def create_rental
```

Comment:
- [ ] In this method you are getting input from users and creating the rental, This method does not follow `single responsibility principle`. Please make sure to separate these operations into different methods to achieve `single responsibility principle`.